### DISCUSSION TITLE
zh/reference/get_started/install_containers

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#454] [Q&A]
# zh/reference/get_started/install_containers

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.edurt.io/zh/reference/get_started/install_containers.html?h=docker

**Comment by bindie2005** [comment]
你好，按上述方法，在docker中部署时，无法启动服务，报以下错误：
2023-10-23 11:45:03 2023-10-23 03:45:03,627 ERROR [main] com.zaxxer.hikari.pool.HikariPool [HikariPool.java:594] HikariPool-1 - Exception during pool initialization.
2023-10-23 11:45:03 com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure
请问，是需要单独部署mysql数据库吗？

**Reply by qianmoQ** [reply]
需要挂载 MySQL 配置，也就是 `application.properties`

**Comment by BloodRaven722** [comment]
请问docker如何挂载MySQL的配置文件？容器内的application.properties路径是什么？

**Reply by Demon1110** [reply]
看了一下，在/opt/app/datacap/configure/application.properties

**Reply by qianmoQ** [reply]
是的挂载到 `/opt/app/datacap/configure/application.properties` 就可以了

**Comment by Demon1110** [comment]
datacap.sql给个单独下载地址吧，Release文件太大了

**Reply by qianmoQ** [reply]
https://github.com/devlive-community/datacap/blob/dev/core/datacap-server/src/main/schema/datacap.sql
